apps/cowswap-frontend/src/modules/trade/containers/TradeWidget/TradeWidgetForm.tsx

```diff
@@ -95,8 +95,8 @@ export function TradeWidgetForm(props: TradeWidgetProps) {
       <styledEl.ContainerBox>
         <styledEl.Header>
           {isAlternativeOrderModalVisible ? <div></div> : <TradeWidgetLinks isDropdown={isInjectedWidgetMode} />}
-          {isInjectedWidgetMode && !injectedWidgetParams.hideConnectButton && (
-            <AccountElement isWidgetMode={isInjectedWidgetMode} pendingActivities={pendingActivity} />
+          {isInjectedWidgetMode && injectedWidgetParams.standaloneMode && (
+            <AccountElement standaloneMode={injectedWidgetParams.standaloneMode} pendingActivities={pendingActivity} />
```

Comment:
Since `standaloneMode` must be `true` to get into this conditional:

```suggestion
            <AccountElement standaloneMode pendingActivities={pendingActivity} />
```